news/627.misc

```diff
@@ -0,0 +1 @@
+Add documentation describing the single source of truth concept and add a diagram for data-flow used in mbed-targets
```

Comment:
`create-news-file` util to the rescue. If you're using latest version of `mbed-tools-ci-scripts` it should be available in the venv. :)

Reply:
Good to know, in this case I was just using the Jira number